### PR TITLE
app-portage/emlop: add 0.6.1

### PR DESCRIPTION
**Pull request by vincentdephily**
Compared to 0.5.0, apart from the new crate versions, I bumped the rust dep to match emlop's MSRV, and moved it from `DEPENDS` to `BDEPENDS`.

I skipped the 0.6.0 release because its test suite was deadlocking when run from portage.

### CODE REVIEW

**Comment by vincentdephily** [comment]
Not sure how long we want to keep the old version in the tree ?
Also, what's the process to get this keyworded stable and on other arches ?

File: app-portage/emlop/emlop-0.6.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+# Dependent crate licenses
+LICENSE+="
+	   MIT Unicode-DFS-2016
+	   || ( Apache-2.0 Boost-1.0 )
+"
+
+SLOT="0"
+KEYWORDS="~amd64"
+
+DEPEND=""
```
Comment (Flowdalic) [inline app-portage/emlop/emlop-0.6.1.ebuild:94]: No reason to declare those variables empty.

```suggestion
DEPEND=""
```

**Comment by Flowdalic** [comment]
> Not sure how long we want to keep the old version in the tree? Also, what's the process to get this keyworded stable and on other arches?

We would hopefully stabilize the first emlop version after it has been in ::gentoo for some time (usually 30 days). See https://devmanual.gentoo.org/keywording/. After emlop got its first stable version https://packages.gentoo.org/maintainer/moltonel@gmail.com/stabilization will show you when it would be usually time for the next.

I found that emlop's metadata.xml does not contain your real name. I wonder if this is intentional since you are using your real name on github?

File: app-portage/emlop/emlop-0.6.1.ebuild (hunk 1 above)
Comment (vincentdephily) [reply app-portage/emlop/emlop-0.6.1.ebuild:94]: Will do for next bump.

**Pull request closed without merging**